Backend/app/rider-platform/rider-app/Main/spec/Storage/ride.yaml

```diff
@@ -20,51 +21,16 @@ imports:
   PaymentIntentId: Kernel.External.Payment.Interface.Types
   AccountId: Kernel.External.Payment.Interface.Types
   Months: Kernel.Types.Time
+  DbHash: Kernel.External.Encryption
 
 Ride:
   tableName: ride
-  derives: "Generic,Show"
+  derives: "Generic"
   types:
     RideStatus:
       enum: "NEW,INPROGRESS,COMPLETED,CANCELLED"
       derive: "HttpInstance"
     BPPRide: {}
-    RideAPIEntity:
```

Comment:
Why did you remove this?

Reply:
api entity shouldn't be in storage file..aslo keeping it here will leads to ambiguity in the beamType